libs/@local/harpc/types/src/version.rs

```diff
@@ -0,0 +1,23 @@
+use core::fmt::{self, Display, Formatter};
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[cfg_attr(feature = "proptest", derive(test_strategy::Arbitrary))]
+pub struct Version {
+    pub major: u8,
+    pub minor: u8,
+}
+
+impl Version {
+    #[must_use]
+    pub const fn new(major: u8, minor: u8) -> Self {
+        Self { major, minor }
+    }
+}
```

Comment:
Fields are public, let's not add a `new` method
```suggestion
```

Reply:
removed in [ded6b3c](https://github.com/hashintel/hash/pull/4283/commits/ded6b3c329c577706ba0c28459744661636d4eee)